### DISCUSSION TITLE
Apera PH Probe

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#966] [REM]
My Atlas Scientific probe was spot on until the last week or so.

I noticed I was using a LOT of acid and checked using reagents.  It's way of--ph is off the scale acid.

I ordered a new set, but they won't get here until next Friday.  Meanwhile, I found an Apera probe for $32 that will be delivered today!

Anyone tried them?

**Comment by johnny2678** [comment]
> Anyone tried them?

Have a link?

**Comment by davthomaspilot** [comment]
https://www.amazon.com/dp/B076BFK5WY?psc=1&ref=ppx_yo2ov_dt_b_product_details

**Reply by johnny2678** [reply]
Nice! Keep us posted
